httomo/runner/loader.py

```diff
@@ -166,12 +167,16 @@ def __init__(
         slicing_dim: Literal[0, 1, 2],
         comm: MPI.Comm,
     ) -> None:
+        if slicing_dim != 0:
+            raise NotImplementedError("Only slicing dim 0 is currently supported")
+
         self._in_file = in_file
         self._data_path = data_path
         self._image_key_path = image_key_path
         self._angles = angles
         self._slicing_dim = slicing_dim
         self._comm = comm
+        self._h5file = h5py.File(in_file, "r", driver="mpio", comm = comm)
```

Comment:
I assume the change to avoid the `mpio` driver for reading is done in the other branch?